test/sequential/test-http-default-timeout-handler.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+const server = http.createServer(common.mustCall((req, res) => {
+  server.close();
+  // Do nothing, just wait for the client socket timeout and close the
+  // connection.
+}));
+
+server.listen(0, common.mustCall(() => {
+  let req;
+  const timer = setTimeout(() => {
+    req.abort();
+    assert.fail('should not timeout here');
+  }, 1000);
```

Comment:
I assume this is just a 'speed-up' of the default test timeout handler, could we just change the timeout for this test then via command-line option (or just leave it as is to avoid unnecessary flakiness)?